testing/test_capture.py

```diff
@@ -344,10 +344,7 @@ def teardown_module(function):
 
     def test_logging_initialized_in_test(self, testdir):
         p = testdir.makepyfile("""
-            import sys
```

Comment:
Hmm I fear if we will reintroduce the bug described in 2e80512bb8a04f141a34fc6586b3518774954193. It was so long ago (issue number 8) that it probably is fine now.

If we decide this is no longer relevant, we should just remove this test then.

Reply:
Who can tell if this is still relevant? Probably the author of the fix (@hpk42)

Reply:
If nobody manifests themselves, I propose just removing the test. I don't see how to keep the test around if we introduce the logging module into the core, after all.